libnavigator/src/test/java/com/mapbox/navigation/navigator/NavigatorMapperTest.kt

```diff
@@ -393,7 +386,7 @@ class NavigatorMapperTest {
     }
 
     @Test
-    fun `unsupported alert types are ignored in the progress update`() {
```

Comment:
Same as above

> Why we removed this test? 🤔

Reply:
because only one unsupported type was Incidents, and from now it's supporting